dev/io.openliberty.microprofile.config.internal_repeat_tests/src/io/openliberty/microprofile/config/fat/repeat/ConfigRepeatActions.java

```diff
@@ -0,0 +1,133 @@
+ *******************************************************************************/
+package io.openliberty.microprofile.config.fat.repeat;
+
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.rules.repeater.MicroProfileActions;
+import componenttest.rules.repeater.RepeatTests;
+
+/**
+ * Contains static methods for creating standard RepeatTests rules for Config tests
+ *
+ * MicroProfile 1.2 = Config 1.1
+ * MicroProfile 1.3 = Config 1.2
+ * MicroProfile 1.4 = Config 1.3
+ * MicroProfile 2.0 = Config 1.3
+ * MicroProfile 2.1 = Config 1.3
+ * MicroProfile 2.2 = Config 1.3
+ * MicroProfile 3.0 = Config 1.3
+ * MicroProfile 3.2 = Config 1.3
+ * MicroProfile 3.3 = Config 1.4
+ * MicroProfile 4.0 = Config 2.0
+ * MicroProfile 4.1 = Config 2.0
+ * MicroProfile 5.0 = Config 3.0
+ * MicroProfile 6.0 = Config 3.0
+ * MicroProfile 6.1 = Config 3.1
+ * MicroProfile 7.0 = Config 3.1
+ */
+public class ConfigRepeatActions {
+
+    /**
+     * Return a rule to repeat tests for MicroProfile 7.0, 6.0, 4.1 and 3.3.
+     * This translates to Config 3.1, 3.0, 2.0 and 1.4 respectively.
+     *
+     * This is the default set of repeats for most Config tests.
+     * It provides good coverage of most code paths.
+     *
+     * @param server the server name
+     * @return the RepeatTests rule
+     */
+    public static RepeatTests repeatDefault(String server) {
+        return MicroProfileActions.repeat(server, TestMode.FULL,
+                                          MicroProfileActions.MP70_EE10, //Config 3.1
+                                          MicroProfileActions.MP70_EE11, //Config 3.1
+                                          MicroProfileActions.MP60, //Config 3.0
+                                          MicroProfileActions.MP41, //Config 2.0
+                                          MicroProfileActions.MP33); //Config 1.4
+    }
+
+    /**
+     * Return a rule to repeat tests for MicroProfile 7.0, 6.0 and 4.1.
+     * This translates to Config 3.1, 3.0 and 2.0 respectively.
+     *
+     * This is the default set of repeats for most Config tests.
+     * It provides good coverage of most code paths.
+     *
+     * @param server the server name
+     * @return the RepeatTests rule
+     */
+    public static RepeatTests repeatDefault20Up(String server) {
+        return MicroProfileActions.repeat(server, TestMode.FULL,
+                                          MicroProfileActions.MP70_EE10, //Config 3.1
+                                          MicroProfileActions.MP70_EE11, //Config 3.1
+                                          MicroProfileActions.MP60, //Config 3.0
+                                          MicroProfileActions.MP41); //Config 2.0
+    }
+
+    /**
+     * Return a rule to repeat tests for MicroProfile 7.0 and 6.0.
+     * This translates to Config 3.1 and 3.0.
+     *
+     * This is the default set of repeats for most Config tests.
+     * It provides good coverage of most code paths.
```

Comment:
Most of the methods in this class say they're the default set of repeats for most config tests, which is a little confusing.

Reply:
Hmmm admittedly a copy-paste oversight